test/fw/ptl/utils/plugins/ptl_test_db.py

```diff
@@ -1612,18 +1613,33 @@ def __init__(self, dbtype, dbpath, dbaccess):
             raise PTLDbError(rc=1, rv=False, msg=_msg)
         elif not self.dbpath.endswith('.json'):
             self.dbpath = self.dbpath.rstrip('.db') + '.json'
+        self._data = {}
         self.jdata = {}
+        self.run_count = 1
```

Comment:
i suggest to rename this to `cur_repeat_count` or `cur_count` (same in PtlTestTestRunner)

Reply:
done